js/util.js

```diff
@@ -44,4 +44,9 @@ const getRandomItems = (items) => {
   return randomItems;
 };
 
-export { getRandomInteger, getRandomFractional, getRandomArrayElement, getRandomItems };
+const isEscEvent = (evt) => {
+  return evt.key === ('Escape' || 'Esc');
```

Comment:
```suggestion
  return evt.key === ('Escape' || 'Esc');
```
Вот это классное место для enum`ов